examples/mqtt/mqtt_viewall.py

```diff
@@ -50,12 +50,12 @@ def on_connect(client, userdata, flags, rc):
 def on_disconnect(client, userdata, rc):
     print('Disconnected!')
 
-def on_message(client, userdata, msg, retain):
+def on_message(client, userdata, msg):
     print('Received on {0}: {1}'.format(msg.topic, msg.payload.decode('utf-8')))
 
 
 # Create MQTT client and connect to Adafruit IO.
-client = mqtt.Client()
+client = mqtt.Client(mqtt.CallbackAPIVersion.VERSION1)
```

Comment:
Can we make the kwarg a `default` argument that we don't need to pass in from user code?

Reply:
This example appears to be using the Paho client directly, instead of our library client, updated comment above to clarify.
Not entirely sure what the examples for except to aid understanding / testing in the debug sense, as it shows the user all data arriving.

Reply:
Updating the comment is OK